tools.go

```diff
@@ -1,6 +1,5 @@
 package tools
 
 import (
-	// Import the v1 API so that we can sync the CRD into the install folder
-	_ "github.com/openshift/api/machine/v1"
```

Comment:
Do we not need this anymore?

Reply:
yes, it's imported directly in the code in several places, like [here](https://github.com/openshift/machine-api-operator/blob/7dd1f5294b1ff9730f51ebad442035938240d94d/pkg/webhooks/machine_webhook.go#L14)